src/instructlab/model/train.py

```diff
@@ -282,6 +282,10 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     is_flag=True,
     help="if true, enables the legacy linux training code path from release 0.17.0 and prior.",
 )
+@click.option(
```

Comment:
can we add some help text? most people will not know how to interpret this

Reply:
if you are looking for this to inherit from the cfg, you'll need the clickext.ConfigOption class on here.

Reply:
Gotcha, I'll make sure this is on it